platformio.ini

```diff
@@ -45,6 +45,7 @@ build_flags =
 	-D OSW_TARGET_PLATFORM_HEADER='"platform/LIGHT_EDITION_V3_2.h"'
 	-D OSW_FEATURE_STATS_STEPS
 	-D OSW_FEATURE_WIFI
+	-D OSW_FEATURE_WIFI_ONBOOT ; DO NOT PUSH THIS CHANGE - THIS IS FOR MY PATCHED SCHEMATIC ONLY!!!
```

Comment:
how about u  update `OSW_FEATURE_WIFI_FALLBACK`?

Reply:
Well, that was an accident... Will remove later